### DISCUSSION TITLE
strapi admin with societio unreal client.

### DISCUSSION THREAD

**Discussion by ssshammi** [strapi-community/plugin-io#59] [General]
hello, I cannot find plugin.js file in config. so i am not sure how this is working. But i basically want to use strap admin as a server. maybe I add singular content from the unreal client. and then in strapi admin see the contents of the message. 

eg. i login from unreal - regular rest call to authenticate. 
Then i initiate a singular content using socket call that says hello.
the strapi admin list hello
I send messages to the client from strapi admin. 

is it possible? can i see this working in  strapi. has someone uploaded a video ?

**Comment by ComfortablyCoding** [comment]
The `plugins.js` file is not generated by default and needs to be added to the config folder for new projects. The config folder is in the root of your strapi project. See https://docs.strapi.io/dev-docs/project-structure

Regarding your use case, this should be possible. Their is no video showcasing this usecase though.
